pages/generate-test-cases.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Card, Grid, Text } from '@mantine/core';
+import Link from 'next/link';
+import React from 'react';
+import { useRecoilValue } from 'recoil';
+import TestCaseEditor from '../components/TestCaseEditor';
+import { measureBundleState } from '../state/atoms/measureBundle';
+import { measurementPeriodStringState } from '../state/selectors/measurementPeriodString';
+import { Edit } from 'tabler-icons-react';
+
+export default function TestCaseEditorPage() {
+  const measurementPeriodString = useRecoilValue(measurementPeriodStringState);
+  const measureBundle = useRecoilValue(measureBundleState);
+  return (
+    <>
+      <Grid>
+        <Grid.Col span={12} sx={() => ({ backgroundColor: 'black', height: 'auto', padding: 20 })}>
+          <div
+            style={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              alignItems: 'center'
+            }}
+          >
+            <Card sx={() => ({ width: 'fit-content' })}>
```

Comment:
Similar to comment about how this page _could_ be navigated to without a measure bundle (only if the user goes awry), maybe we only render this card if the measure bundle/period is not null. This goes in tandem with adding helpful text instead of the test case editor for navigating back to the homepage